src/open_inwoner/accounts/views/auth_oidc.py

```diff
@@ -97,16 +97,34 @@ def get_success_url(self):
 
     def get(self, request):
         assert self.config_class is not None
+        config = self.config_class.get_solo()
 
-        if id_token := request.session.get("oidc_id_token"):
-            config = self.config_class.get_solo()
-            do_op_logout(config, id_token)
+        if not (logout_endpoint := config.oidc_op_logout_endpoint):
+            logger.warning("No OIDC logout endpoint defined")
```

Comment:
You do another check `if logout_endpoint` at l 113. So instead of checking here, you could do `logout_endpoint = config.oidc_op_logout_endpoint` at l 112, continue with `if logout_endpoint`, and add the `logger.warning` outside the conditional right before the response.